backend/src/test/java/com/woowacourse/kkogkkog/acceptance/CouponAcceptanceTest.java

```diff
@@ -19,6 +19,7 @@
 import io.restassured.RestAssured;
 import io.restassured.response.ExtractableResponse;
 import io.restassured.response.Response;
+import java.time.LocalDate;
 import java.util.List;
```

Comment:
잘 안 보이기는 하는데, 안쓰이지 않나요?

```suggestion
import java.util.List;
```